test/bpf/Makefile

```diff
@@ -58,4 +63,4 @@ unit-tests: $(ALL_TESTS)
 clean:
 	@$(ECHO_CLEAN)
 	-$(QUIET)rm -f $(TARGETS)
-
+	-$(QUIET)rm -r ../../bpf/mock/mocks
```

Comment:
nit: Maybe we should make this a clean target in `bpf/mock` and just call `$(MAKE) -C ../../bpf/mock clean` here? Then the cleanup Makefile logic is next to the other logic, in case we need to make any changes in future.

Reply:
Done.